src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -1699,41 +1701,55 @@ ChildNodeList(MultiNode *multiNode)
 
 /*
  * UniqueJobId allocates and returns a unique jobId for the job to be executed.
- * This allocation occurs both in shared memory and in write ahead logs; writing
- * to logs avoids the risk of having jobId collisions.
- *
- * Please note that the jobId sequence wraps around after 2^32 integers. This
```

Comment:
Maybe, adding a one-line comment about how the final jobid looks like could help the readers.